pages/components/Sidebar.js

```diff
@@ -35,6 +35,7 @@ export const Sidebar = ({ onClick }) => (
       <MenuItem onClick={onClick} href="/language" title="Language and text" />
       <MenuItem onClick={onClick} href="/navigation" title="Navigation" />
       <MenuItem onClick={onClick} href="/animation" title="Animation" />
+      <MenuItem onClick={onClick} href="/microcopy" title="Microcopy" />
```

Comment:
Not important, but I'd put this next to language above, as they're slightly related

Reply:
Sure, happy to move it